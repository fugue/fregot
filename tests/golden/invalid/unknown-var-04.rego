package fregot.tests.invalid.unkown_var_04

test_double {
  double(x, y)
  y == 2
}
