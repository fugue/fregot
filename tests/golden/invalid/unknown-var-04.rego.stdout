passed: 0, failed: 0, errored: 1
fregot.tests.invalid.unkown_var_04.test_double:
fregot ([34meval error[0m):
  "unknown-var-04.rego" (line 4, column 3):
  unknown function:

    4|   double(x, y)
         [31m^^^^^^^^^^^^[0m

  Unknown function call: double
